rest_framework_simplejwt/backends.py

```diff
@@ -50,7 +50,12 @@ def encode(self, payload):
             jwt_payload['iss'] = self.issuer
 
         token = jwt.encode(jwt_payload, self.signing_key, algorithm=self.algorithm)
-        return token.decode('utf-8')
+        if isinstance(token, str):
+            # For PyJWT >= 2.0.0a1
+            return token
+        elif isinstance(token, bytes):
+            # For PyJWT <= 1.7.1
+            return token.decode('utf-8')
```

Comment:
```suggestion
        if isinstance(token, bytes):
            # For PyJWT <= 1.7.1
            return token.decode('utf-8')
        # For PyJWT >= 2.0.0a1
        return token
```
Need to make sure that SOMETHING is returned to at least raise an error if None is being decoded.